ci-operator/config/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-release-4.6.yaml

```diff
@@ -61,6 +61,10 @@ tests:
   commands: make generate-check
   container:
     from: src
+- as: e2e-agnostic
+  steps:
+    cluster_profile: gcp
+    workflow: openshift-e2e-gcp
```

Comment:
You could omit 4.6, CBO won't be part of OCP until 4.7. Not a big deal having the test enabled but it shouldn't be used anymore